src/types/theme.ts

```diff
@@ -0,0 +1,189 @@
+/**
+ * Copyright 2020, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+enum Breakpoint {
+  untilKilo = 'untilKilo',
+  kilo = 'kilo',
+  kiloToMega = 'kiloToMega',
+  mega = 'mega',
+  untilMega = 'untilMega',
+  megaToGiga = 'megaToGiga',
+  giga = 'giga',
+  gigaToTera = 'gigaToTera',
+  tera = 'tera',
+  afterTera = 'afterTera'
+}
+
+type Typography = {
+  fontSize: string;
+  lineHeight: string;
+};
+
+export interface Theme {
```

Comment:
You can still break these by groups and use `Merge` if you like.

Reply:
Is `Merge` a utility type? I couldn't find any documentation for it.